tests/run-make/metadata-flag-frobs-symbols/rmake.rs

```diff
@@ -0,0 +1,21 @@
+// In this test, foo.rs is compiled twice with different hashes tied to its
+// symbols thanks to the metadata flag. bar.rs then ensures that the memory locations
+// of foo's symbols are different even though they came from the same original source code.
+// This checks that the metadata flag is doing its job.
+// See https://github.com/rust-lang/rust/issues/14471
+
+//@ ignore-cross-compile
+
+use run_make_support::{run, rust_lib_name, rustc};
+
+fn main() {
+    rustc().input("foo.rs").metadata("a").extra_filename("-a").run();
+    rustc().input("foo.rs").metadata("b").extra_filename("-b").run();
+    rustc()
+        .input("bar.rs")
+        .extern_("foo1", rust_lib_name("foo-a"))
+        .extern_("foo2", rust_lib_name("foo-b"))
+        .print("link-args")
+        .run();
```

Comment:
Question: is this supposed to be checking otuput of `--print=link-args`? I know the original test does the same thing.

Reply:
You mean, if it should be `run_unchecked`? I don't see why, if the original test worked on all platforms as `run`, why would we need to reduce the exactitude of the test by not checking?

Reply:
No I mean that this invocation uses `--print=link-args` but we're not checking what stdout contains (or what the content of `--print=link-args` is), is that intentional?

Reply:
That is quite weird indeed, my guess is that the test writer left it in as a way to debug their own test if it failed. The test works locally without it, so I am removing it now.

Reply:
Thanks, r=me when CI passes